photos/db.go

```diff
@@ -31,7 +31,7 @@ const (
 	photosSchema = `
 CREATE DATABASE IF NOT EXISTS photos;
```

Comment:
Sort of strange that you're creating the `photos` database here while also requiring it in the URL. Perhaps you should pull out the database out of the supplied URL and ensure that database exists.